integration-libs/opf/payment/root/model/opf-payment.model.ts

```diff
@@ -42,6 +42,7 @@ export interface GlobalOpfPaymentMethods {
   throwPaymentError?(errorOptions?: ErrorDialogOptions): void;
   startLoadIndicator?(): void;
   stopLoadIndicator?(): void;
+  scriptReady?(scriptIdentifier: string): void;
```

Comment:
If we need this property only for CTA, we should use typescript augmentation, and add this object from CTA sub-lib.

See example: `integration-libs/opf/cta/opf-api/model/opf-api-cta-endpoints.model.ts`

Reply:
Would it be better if we avoid to call GlobalService chunck?
that would be a small part of  OpfGlobalService to copy into cta lib